Documentation/Configuration/FlagFiles/Index.rst

```diff
@@ -0,0 +1,108 @@
+.. include:: /Includes.rst.txt
+
+.. _flag-files:
+
+==========
+Flag files
+==========
+
+TYPO3 uses a set of special files known as flag files or indicator files to
+control and manage low-level configurations, behaviors, and security settings
+of the system. These files act as triggers that enable or disable specific
+features or functionalities in TYPO3, often without requiring direct
+modifications to the core configuration files.
+
+Flag files are typically placed in specific locations within the TYPO3 file
+system and are usually named in a way that reflects their purpose.
+
+Below is a list of commonly used TYPO3 flag files, along with explanations of
+their functions and typical use cases.
+
+..  confval-menu::
+    :display: table
+    :Location:
+    :Legacy Location:
+
+    ..  confval:: LOCK_BACKEND
+        :name: flag-file-lock-backend
+        :Content: empty or arbitrary text file
+        :Location: typo3conf/LOCK_BACKEND
```

Comment:
For "main" this is now set to `var/lock/` (composer mode) or `config/` (legacy mode), not `typo3conf` anymore.
See Forge #104126

Reply:
Yes, see my description:  "This is a preparation for https://github.com/TYPO3-Documentation/Changelog-To-Doc/issues/987"

So I first document the old way, backport it, then apply the changes